DS-Session4-Question1/src/meta/utility/UniqueEmployee.java

```diff
@@ -0,0 +1,23 @@
+package meta.utility;
+
+import java.util.HashSet;
+import java.util.List;
+import java.util.Set;
+
+public class UniqueEmployee {
```

Comment:
In order to avoid duplicate entry, we should use a collection which avoid duplicacy. There is no need to create this class.